integration-test/src/test/scala/org/apache/spark/deploy/k8s/integrationtest/KubernetesSuite.scala

```diff
@@ -291,14 +313,21 @@ private[spark] object KubernetesSuite {
   val CONTAINER_LOCAL_SPARK_DISTRO_EXAMPLES_JAR: String = s"local:///opt/spark/examples/jars/" +
     s"${SPARK_DISTRO_EXAMPLES_JAR_FILE.getName}"
   val SPARK_PI_MAIN_CLASS: String = "org.apache.spark.examples.SparkPi"
+  val SPARK_PAGE_RANK_MAIN_CLASS: String = "org.apache.spark.examples.SparkPageRank"
 
   val TEST_SECRET_NAME = "test-secret"
   val TEST_SECRET_KEY = "test-key"
   val TEST_SECRET_VALUE = "test-data"
   val TEST_SECRET_MOUNT_PATH = "/etc/secrets"
 
+  val FILE_DOWNLOAD_PATH = "/var/spark-data/spark-files"
```

Comment:
Is this local to pod or container? Or is this local to integration test? Maybe indicate that in the name?

Reply:
This is the path in the pod where the init-container downloads files to.

Reply:
Renamed.